folly/Traits.h

```diff
@@ -374,7 +374,7 @@ struct Disjunction<T, TList...>
     : std::conditional<T::value, T, Disjunction<TList...>>::type {};
 
 template <typename T>
-struct Negation : std::integral_constant<bool, !T::value> {};
+using Negation = std::integral_constant<bool, !T::value>;
```

Comment:
This should be a struct, not an alias. It should be possible to name the type `Negation<T>` where `T` is an incomplete type.